code/core/src/components/components/syntaxhighlighter/syntaxhighlighter.tsx

```diff
@@ -22,6 +22,7 @@ import tsx from 'react-syntax-highlighter/dist/esm/languages/prism/tsx';
 import typescript from 'react-syntax-highlighter/dist/esm/languages/prism/typescript';
 import yml from 'react-syntax-highlighter/dist/esm/languages/prism/yaml';
 import ReactSyntaxHighlighter from 'react-syntax-highlighter/dist/esm/prism-light';
+import vue from 'react-syntax-highlighter/dist/esm/languages/hljs/vue';
```

Comment:
This import doesn't seem to exist (see the build failure.

Reply:
Oh that is my fault, I assumed it uses highlightjs